ENGAGEHF/ENGAGEHFStandard.swift

```diff
@@ -57,13 +57,28 @@ actor ENGAGEHFStandard: Standard, EnvironmentAccessible, OnboardingConstraint, A
             return Storage.storage().reference().child("users/\(details.accountId)")
         }
     }
+    
+    private var hkSampleMapping: HKSampleMapping {
+        var sampleMapping = HKSampleMapping.default
+        var bodyMassMapping = sampleMapping.quantitySampleMapping[HKQuantityType(.bodyMass)]
+        bodyMassMapping?.unit = MappedUnit(
+            hkunit: .gramUnit(with: .kilo),
+            unit: "kg",
+            system: URL(string: "http://unitsofmeasure.org")!,
+            code: "kg"
+        )
+        sampleMapping.quantitySampleMapping[HKQuantityType(.bodyMass)] = bodyMassMapping
+        
+        return sampleMapping
+    }
```

Comment:
I think it would be good to by-default store in KG to ideally use SI units where possible. But both the app and the server should understand all units.